app/templates/pains_verify_and_coefficient_use.html

```diff
@@ -1,120 +1,143 @@
+
+
 {% extends 'bootstrap/base.html' %}
 {% block title %}
 {{ title }}
 {% endblock %}
 {% block content %}
 <div class="container">
-  <div class="text-center"> 
-    <h1> {{ title }}</h1>
-    <h3>Verify Pains Filter</h3> 
-
-    <form action="\verify_pains_by_error" method=POST enctype=multipart/form-data>
+<div class="text-center">
+   <h1> {{ title }}</h1>
+   <h3>Verify Pains Filter</h3>
+   <form action="\verify_pains_by_error" method=POST enctype=multipart/form-data>
```

Comment:
I don't know that we need to change all of the formatting here.

Reply:
VSCode changed some of the formatting, I have changed it back in latest commit